src/transformers/pipelines/text2text_generation.py

```diff
@@ -142,42 +131,41 @@ def __call__(
 
 
 @add_end_docstrings(PIPELINE_INIT_ARGS)
-class TranslationPipeline(Pipeline):
+class SummarizationPipeline(Text2TextGenerationPipeline):
     """
-    Translates from one language to another.
+    Summarize news articles and other documents.
 
-    This translation pipeline can currently be loaded from :func:`~transformers.pipeline` using the following task
-    identifier: :obj:`"translation_xx_to_yy"`.
+    This summarizing pipeline can currently be loaded from :func:`~transformers.pipeline` using the following task
+    identifier: :obj:`"summarization"`.
 
-    The models that this pipeline can use are models that have been fine-tuned on a translation task. See the
-    up-to-date list of available models on `huggingface.co/models
-    <https://huggingface.co/models?filter=translation>`__.
+    The models that this pipeline can use are models that have been fine-tuned on a summarization task, which is
+    currently, '`bart-large-cnn`', '`t5-small`', '`t5-base`', '`t5-large`', '`t5-3b`', '`t5-11b`'. See the up-to-date
+    list of available models on `huggingface.co/models <https://huggingface.co/models?filter=summarization>`__.
 
     Usage::
-        en_fr_translator = pipeline("translation_en_to_fr")
-        en_fr_translator("How old are you?")
-    """
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+        # use bart in pytorch
+        summarizer = pipeline("summarization")
+        summarizer("Sam Shleifer writes the best docstring examples in the whole world.", min_length=5, max_length=20)
```

Comment:
Can we take this PR as an opportunity to change this docstring. Didn't catch it when this was merged, but I'd like to avoid having an (ex now) person of the team mentioned by name in the examples (or any person at all for that matter).